src/components/Chat/ChatMessageComponent.tsx

```diff
@@ -72,19 +79,17 @@ const ChatMessageComponent = (props: IProps) => {
     );
   } else {
     return (
-      <div>
-        <div className="chat-message">
-          <p className={`chat-message--author-${authorType}`}>
-            <span>{authorType === 'self' ? 'Deg' : props.message.author}</span>,
-            kl.{' '}
-            <span>
-              {props.message.datetime &&
-                timeStringFromDate(props.message.datetime)}
-            </span>
-          </p>
-          {renderMessage()}
-        </div>
-        {RenderFiles()}
+      <div className="chat-message">
+        <img height={10} width={10} src={props.message.imgUrl} />
+        <p className={`chat-message--author-${authorType}`}>
+          <span>{authorType === 'self' ? 'Deg' : props.message.author}</span>,
```

Comment:
```suggestion
          <span>{authorType === 'self' ? 'Deg' : author}</span>,
```
Du har allerede descructa `props.message`